submitter/cli.py

```diff
@@ -87,3 +89,22 @@ def create_queue(name):
     """Create queue with NAME supplied as argument"""
     queue = create(name)
     logger.info(queue.url)
+
+
+@main.command()
+def verify_dspace_connection():
+    client = DSpaceClient(CONFIG.DSPACE_API_URL, timeout=CONFIG.DSPACE_TIMEOUT)
```

Comment:
I don't think this needs addressing now, as it's currently a topic of active discussion, but I think the `config.Class` might benefit from raising exceptions if required env vars are not present.

When I attempted to run the following, to test this new command:
```shell
WORKSPACE=dev pipenv run submitter verify-dspace-connection
```

I receieved this
```python
...
File "<PATH>/dspace-submission-service/submitter/cli.py", line 96, in verify_dspace_connection
  client = DSpaceClient(CONFIG.DSPACE_API_URL, timeout=CONFIG.DSPACE_TIMEOUT)
File "<VENV>/lib/python3.9/site-packages/dspace/client.py", line 39, in __init__
  self.base_url: str = base_url.rstrip("/")
AttributeError: 'NoneType' object has no attribute 'rstrip'
```

I can see quite clearly, after the fact, that the README indicates these need to be set.  But I think this gets at a subtle question we may want to answer: what if _all_ env vars don't need to be set for certain actions?  Is it possible for these error messages to indicate what's missing?

Again, not requesting changes here, just noting for our ongoing discussions.

Reply:
Great point that I'll consider with work I'm doing on `wiley-deposits`